data_steward/resource_files/fields/fitbit/activity_summary.json

```diff
@@ -0,0 +1,80 @@
+[
```

Comment:
I also added all the schema for the FitBit tables, should we keep yours or mine?

Reply:
@dcampbell-vumc and @nishanthpp93 You've added these in different locations.  They won't conflict, but we should choose one location instead of two.  I thought we had decided on `curation/data_steward/resource_files/fields/wearables/fitbit`?

Reply:
Ah, right. I didn't notice the path. I have mine located in `/wearables/fitbit`

Reply:
I forgot the wearables part. We can use yours